apps/livreGUI/tfEditor/TransferFunctionEditor.cpp

```diff
@@ -66,6 +69,9 @@ TransferFunctionEditor::TransferFunctionEditor( livre::Controller& controller,
     connect( ui->btnConnect, SIGNAL( pressed()), this, SLOT( _connect( )));
     connect( ui->btnDisconnect, SIGNAL( pressed()), this, SLOT( _disconnect()));
 
+    connect( this, &TransferFunctionEditor::transferFunctionChanged,
+             this, &TransferFunctionEditor::_onTransferFunctionChanged );
```

Comment:
Imo Qt does this automatically if connection type is AUTO and signaler is in different thread than the receiver.